src/connect/components/tree-view-item/tree-view-item.tsx

```diff
@@ -70,19 +73,19 @@ export const DefaultOptions = [
 
 export type DefaultOptionId = (typeof DefaultOptions)[number]['id'];
 
-export interface ConnectTreeViewItemProps<T extends string = DefaultOptionId>
+export interface ConnectTreeViewItemProps
     extends Pick<
         TreeViewItemProps,
         'children' | 'onClick' | 'buttonProps' | 'level'
     > {
-    item: TreeItem<T>;
-    onDropEvent?: UseDraggableTargetProps<TreeItem<T>>['onDropEvent'];
-    onDropActivate?: (dropTargetItem: TreeItem<T>) => void;
-    defaultOptions?: ConnectDropdownMenuItem<T>[];
-    onOptionsClick?: (item: TreeItem<T>, option: T) => void;
+    item: TreeItem;
+    onDropEvent?: UseDraggableTargetProps<TreeItem>['onDropEvent'];
+    onDropActivate?: (dropTargetItem: TreeItem) => void;
+    defaultOptions?: ConnectDropdownMenuItem[];
+    onOptionsClick?: (item: TreeItem, option: DefaultOptionId) => void;
```

Comment:
I removed the generics to return directly the `DefaultOptionId` instead

Reply:
this will force you to use the `DefaultOptionId` const type, so you won't be able to use other strings for the `option` param

Reply:
fixed: [4b1a171](https://github.com/powerhouse-inc/design-system/pull/94/commits/4b1a171e9b9f348177a8710eb0113bfc32a737ad)